### PR TITLE
[Review] Request from 'schubi2' @ 'yast/yast-registration/review_171108_warning_text_changed'

### PR DESCRIPTION
**Pull request by schubi2**
Please review the following changes:
  * 682fea8 changed warning text if user has skipped registration

### CODE REVIEW

File: src/lib/registration/ui/base_system_registration_dialog.rb [hunk 1]
```diff
@@ -359,10 +359,12 @@ def help_text
       # @return [Boolean] true when skipping has been confirmed
       def show_skipping_warning
         # Popup question: confirm skipping the registration
-        warning = _("If you do not register your system we will not be able\n" \
-            "to grant you access to the update repositories.\n\n" \
-            "You can register after the installation or visit our\n" \
-            "Customer Center for online registration.")
+        warning = _("Without registration, update-channels will not be\n" \
```
Comment (mvidner) [inline src/lib/registration/ui/base_system_registration_dialog.rb]: "update channels" without the hyphen

Comment (lslezak) [inline src/lib/registration/ui/base_system_registration_dialog.rb]: I'm not sure about the wording here... @mvidner ?

File: src/lib/registration/ui/base_system_registration_dialog.rb [hunk 2]
```diff
@@ -359,10 +359,12 @@ def help_text
...
+            "configured. This disables updates and security fixes.\n\n" \
+            "A full system can be installed using the\n" \
+            "SLE-15-Packages media from download.suse.com.\n" \
```
Comment (mvidner) [inline src/lib/registration/ui/base_system_registration_dialog.rb]: "media" is a plural. The singular is "medium".
Is the medium name correct?
How stable is it? Should we insert it with `format`?

File: src/lib/registration/ui/base_system_registration_dialog.rb [hunk 3]
```diff
@@ -359,10 +359,12 @@ def help_text
...
+            "Without this media only a minimum system is available\n" \
```
Comment (mvidner) [inline src/lib/registration/ui/base_system_registration_dialog.rb]: "medium"

Comment (schubi2) [reply src/lib/registration/ui/base_system_registration_dialog.rb]: Hm, I have asked a native speaker to check these strings and she has approved it.

File: src/lib/registration/ui/base_system_registration_dialog.rb (hunk 2 above)
Comment (lslezak) [reply src/lib/registration/ui/base_system_registration_dialog.rb]: The medium name is correct. However I would not hard code it here but rather use a string interpolation `%{media_name}` placeholder as it still can be changed. The same with the site name.

That allows us to update the names without breaking the translations.

BTW I'd probably prefer `https://download.suse.com` value so it's clear what does it mean.

File: src/lib/registration/ui/base_system_registration_dialog.rb (hunk 3 above)
Comment (lslezak) [reply src/lib/registration/ui/base_system_registration_dialog.rb]: It's a bit complicated. It's actually a set of 3 media, but in reality just the DVD1 is enough (DVD2 = sources, DVD3 = debug packages IIRC).

Comment (mvidner) [reply src/lib/registration/ui/base_system_registration_dialog.rb]: OK, if we are talking about several pieces of storage media, then "these media".

File: src/lib/registration/ui/base_system_registration_dialog.rb [hunk 4]
```diff
@@ -359,10 +359,16 @@ def help_text
...
       def show_skipping_warning
         # Popup question: confirm skipping the registration
-        warning = _("If you do not register your system we will not be able\n" \
-            "to grant you access to the update repositories.\n\n" \
-            "You can register after the installation or visit our\n" \
-            "Customer Center for online registration.")
+        # TRANSLATORS:
+        # %{media_name} is the media name (e.g. SLE-15-Packages),
+        # %{download_url} is an URL link (e.g. https://download.suse.com)
+        warning = _("Without registration, update-channels will not be\n" \
```
Comment (mvidner) [inline src/lib/registration/ui/base_system_registration_dialog.rb]: "update channels" without the hyphen

**Pull request merged**